### DISCUSSION TITLE
Openshift support?

### DISCUSSION THREAD

**Discussion by APCG** [bentoml/BentoML#999] [General]
Hi all, do you know if BentoML supports openshift deployment? Has anybody tried it and can share some resource? Your help will be much appreciated. :)

Thanks!

**Comment by parano** [comment]
Hi @APCG 

There isn't an automated deployment operator for Openshift cluster in BentoML yet(like the ones we built for AWS sagemaker, Lambda, and Azure ML).  Although it is possible to deploy the API server docker image generated with BentoML to an Openshift cluster fairly easily and it should be very similar to deploying BentoML API server to Kubernetes, you may use this doc as a reference: https://docs.bentoml.org/en/latest/deployment/kubernetes.html

If anyone in the community is interested in writing a deployment guide for OpenShift that'd be highly appreciated!
